### PR TITLE
[usbutils] Update to 014. JB#56442

### PR DESCRIPTION
**Pull request by pvuorela**
Devel subpackage deleted, the .pc file doesn't exist anymore.

--disable-usbids and related parts also removed upstream.
lsusb.py now having /usr/share/hwdata/usb.ids support upstream.
-> no need for patching.

As side-note the .py version had been effectively broken long
time as old version was using python2.

Changes file had some invalid entries and it's not mostly
our history anyway -> deleted.

@mlehtima @Tomin1 @Thaodan

### CODE REVIEW

File: rpm/usbutils.spec [hunk 1]
```diff
@@ -19,44 +15,22 @@ BuildRequires:  libtool
...
+%license LICENSES/GPL-2.0-only.txt
 %{_bindir}/*
-
-%files devel
-%defattr(-,root,root,-)
-%doc AUTHORS COPYING ChangeLog NEWS README
-%{_datadir}/pkgconfig/usbutils.pc
-%{_mandir}/*/*
-
+%exclude %{_mandir}/*/*
```
Comment (Thaodan) [inline rpm/usbutils.spec:36]: That could go to a doc subpackage.
See JB#55515.

**Review by Thaodan — APPROVED** [review]

File: rpm/usbutils.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/usbutils.spec:36]: That's mostly about api documentation, html and such. Man pages on mobile devices are not really that useful.

Comment (Thaodan) [reply rpm/usbutils.spec:36]: This would have been for SDK not for device.

Comment (pvuorela) [reply rpm/usbutils.spec:36]: Then again usb stuff on sdk, hm.

**Pull request merged**